xDrip/Scenes/Settings/SettingsUserType/SettingsUserTypeRoot/SettingsUserTypeRootViewController.swift

```diff
@@ -49,11 +49,15 @@ class SettingsUserTypeRootViewController: NibViewController, SettingsUserTypeRoo
     }
     
     // MARK: IB
+    @IBOutlet private weak var segmentedControl: UISegmentedControl!
+    @IBOutlet private weak var containerView: UIView!
     
+    private var tabBar: UITabBarController?
```

Comment:
Let's rename it to `embeddedTabBarController`, because `tabBar` looks like a view.